sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -608,15 +608,21 @@ def test_command_component_get_latest_label(self, client: MLClient, randstr: Cal
     def test_anonymous_registration_from_load_component(self, client: MLClient, randstr: Callable[[str], str]) -> None:
         command_component = load_component(source="./tests/test_configs/components/helloworld_component.yml")
         component_resource = client.components.create_or_update(command_component, is_anonymous=True)
-        assert component_resource.name == ANONYMOUS_COMPONENT_NAME
+        assert component_resource.name == command_component.name
```

Comment:
Do we have a job link or component link which I can give it a test?

Reply:
I tried but it seems I still get azureml_anonymous

https://ml.azure.com/runs/stoic_parang_24142q7jlz?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourcegroups/sdk/workspaces/sdk-master&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Reply:
please check this: https://ml.azure.com/runs/lemon_muscle_mmm3hx9zr7?wsid=/subscriptions/74eccef0-4b8d-4f83-b5f9-fa100d155b22/resourcegroups/AmlComponentNotebook/workspaces/SDK-Notebook-Stable-DEV-20230206&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#